src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -74,7 +74,8 @@
         "symfony/translation": "<4.2",
         "symfony/twig-bridge": "<4.1.1",
         "symfony/validator": "<4.1",
-        "symfony/workflow": "<4.1"
+        "symfony/workflow": "<4.1",
+        "symfony/yaml": "<=4.2"
```

Comment:
`<4.2`

Reply:
Done, but still:
> - Installing symfony/yaml (4.2.x-dev): Downloading (100%)

Reply:
which mean it worked actually

Reply:
This is the fails before the last commit https://travis-ci.org/symfony/symfony/jobs/430972569#L2994